excercises/transformation/SDLProject/solution/solution.cpp

```diff
@@ -0,0 +1,158 @@
+#define GL_SILENCE_DEPRECATION
```

Comment:
Solution should _not_ be available to students. Please remove this file from the commit for now (thru `.gitignore` so as to not lose its contents).